src/Application.php

```diff
@@ -63,12 +64,14 @@ public function run(): void
      *
      * The resulting middleware, in both cases, is piped to the pipeline.
      *
-     * @param string|array|callable|MiddlewareInterface|RequestHandlerInterface $middlewareOrPath
+     * @param non-empty-string|array|callable|MiddlewareInterface|RequestHandlerInterface $middlewareOrPath
```

Comment:
`$path` for Route or RouteCollector is now `non-empty-string`, so mostly we'd have to assert non empty or suppress all the issues - it can be expanded to string for this method specifically though because Stratigility accepts `string`